src/dapp/app.ts

```diff
@@ -0,0 +1,21 @@
+import {
+    returnLocationCoordiantes,
+    returnEntiretyOfEvents,
+    Event_shadow_coordinates,
+} from './utils/api_request_helpers'
+
+import dotenv from 'dotenv'
+
+// Load environment variables from .env file
+dotenv.config()
```

Comment:
Since node v20 you can load env files directly with `node --env-file=.env app.js`.

Reply:
Initialize your Node.js application with predefined configurations using the following CLI command:
node --env-file=config.env index.js. I need to run a command. I don't know how yet. Might change in a future version.